pkg/tracking/store/sql/experiments.go

```diff
@@ -134,6 +134,62 @@ func (s TrackingSQLStore) DeleteExperiment(id string) *contract.Error {
 	return nil
 }
 
+func (s TrackingSQLStore) RestoreExperiment(id string) *contract.Error {
+	idInt, err := strconv.ParseInt(id, 10, 32)
+	if err != nil {
+		return contract.NewErrorWith(
+			protos.ErrorCode_INVALID_PARAMETER_VALUE,
+			fmt.Sprintf("failed to convert experiment id (%s) to int", id),
+			err,
+		)
+	}
+
+	if err := s.db.Transaction(func(transaction *gorm.DB) error {
+		// Update experiment
+		uex := transaction.Model(&models.Experiment{}).
```

Comment:
I believe the Python implementation will only update when the experiment has `ViewType.DELETED_ONLY`. In the Go implementation you could restore an experiment that was not deleted? Or is this checked elsewhere?